packages/react-button/src/components/Button/useButtonClasses.ts

```diff
@@ -114,6 +114,23 @@ export const useButtonClasses = makeVariantClasses<ButtonState>({
           marginLeft: 'var(--button-contentGap)',
         },
 
+        '@media (forced-colors: active)': {
+          forcedColorAdjust: 'var(--button-forcedColorAdjust)',
+
+          background: 'var(--button-highContrast-background)',
+          borderColor: 'var(--button-highContrast-borderColor)',
+          color: 'var(--button-highContrast-contentColor)',
+
+          ':global(.ms-Fabric--isFocusVisible) &:focus::after': {
+            borderColor: 'var(--button-highContrast-focusColor)',
+            boxShadow: '0 0 0 var(--button-focusInnerWidth, 1px) var(--button-highContrast-focusInnerColor) inset',
+          },
```

Comment:
this is not quite right; it will disregard the media query

Reply:
Fixed!